kong/kong_conformance_test.go

```diff
@@ -0,0 +1,28 @@
+package kong_test
+
+import (
+	"testing"
+
+	"github.com/alecthomas/kong"
+
+	"foxygo.at/jsonnext"
+	"foxygo.at/jsonnext/conformance"
+	jxkong "foxygo.at/jsonnext/kong"
+)
+
+type suite struct{}
```

Comment:
This confused me a little. Why did you call this type suite, rather than kongParser or similar?

Reply:
well, it is not a kong parser but a "kong parser conformance test suite". It is in a file called "kong_conformance_test.go" so "~kong~ parser ~conformance test~ suite". It implements `CLIParser`, so that leaves suite.

I will ponder on a better name. (I wasn't entirely happy with it myself - i went through a bunch of names for all the different parts of the conformance test - and I actually meant to rename `clitest.go` to `conformance.go` but forgot, and I'm not so sure now)